net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+wireless-tools +ethtool +procps +CONFIG_PCI_SUPPORT:pciutils +CONFIG_USB_SUPPORT:usbutils
```

Comment:
that's fine, but most of these deps are not correct for airmon-ng in rc1 so it doesn't make sense to merge this.  since rc3 has other issues (which I believe are resolved in latest svn) we should wait for rc4 or take the relevant patches from upstream into rc3